artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerSessionImpl.java

```diff
@@ -159,7 +160,7 @@
 
    private final SimpleString managementAddress;
 
-   protected final RoutingContext routingContext = new RoutingContextImpl(null);
+   protected final RoutingContext _routingContext;
```

Comment:
we can use just `routingContext` name, dropping `_`

Reply:
@franz1981 I don't follow you

Reply:
@franz1981 I reverted the change that's not needed any longer. 


all amended.